Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -465,6 +465,29 @@ ShaderCode GenerateVertexShaderCode(APIType api_type, const vertex_shader_uid_da
   // get rasterized correctly.
   out.Write("o.pos.xy = o.pos.xy - o.pos.w * " I_PIXELCENTERCORRECTION ".xy;\n");
 
+  if (g_ActiveConfig.bVertexRounding)
```

Comment:
You will also need to set this in the shader uuid. Otherwise the setting won't toggle on and off as expected, each shader be stuck with the setting that was enabled when the shader was genenerated